internal/pkg/config/testdata/test-config-v2.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: config/v2
+kind: config
+
+metadata:
+  validateConfig: true
+  scope:
+    skip:
+    - imports: testing
+    - package: my/special/pkg
+
+source:
+- value:
+    id: "Core Source"
+    package: ^k8s.io/client-go/rest$
+    type: ^TLSClientConfig$
+    field: Password
+- value:
+    # `levee:”source”` should be default, but additional tags
+    # should be configurable
+    fieldtags:
+      - myTag: source
+      - myTag: otherSource
+- call:
+    id: "sensitive environment information"
+    package: os
+    function: getEnv
+    arguments:
+    - value:
+        position: 0
+        const: CFSSL_CA_PK_PASSWORD
+
+sink:
+- call:  &println-sink
+    id: "don't print"
+    package: fmt
+    function: Println
+    unless:
+      - context:
+          package: my/sanctioned/package
+- value:
+    id: "Sink function member"
```

Comment:
I'm not sure using a value specifier here is entirely intuitive. I think a dedicated `funcField` matcher would be easier to understand.

Reply:
I don't know.  On the one hand, yes, functions feel separate from values.  But on the other hand, we're already assuming a good bit of familiarity with Go from anyone building a config.  That functions are first-class objects is part of that familiarity.

I feel like this ends up being a documentation issue, where we should be explicit that any struct field would need to be identified as a `value`.

I'll come up with some more concrete examples for discussion, and see how it feels then.